xbmc/interfaces/python/PythonInvoker.cpp

```diff
@@ -147,7 +147,12 @@ bool CPythonInvoker::execute(const std::string &script, const std::vector<std::s
 
   // copy the arguments into a local buffer
   m_argc = arguments.size();
-  m_argv = new char*[m_argc];
+
+  if(m_argc > 0)
+  {
+    m_argv = new char*[m_argc];
+  }
```

Comment:
Is it OK if I keep them? I've seen people in the past attempt to add new lines to conditional blocks without braces so I try to add braces as a rule.